gc/base/MemorySubSpaceSemiSpace.cpp

```diff
@@ -1114,5 +1114,21 @@ MM_MemorySubSpaceSemiSpace::mergeLargeObjectAllocateStats(MM_EnvironmentBase *en
 	_largeObjectAllocateStats->mergeCurrent(_memorySubSpaceSurvivor->getMemoryPool()->getLargeObjectAllocateStats());
 }
 
+uintptr_t
+MM_MemorySubSpaceSemiSpace::releaseFreeMemoryPages(MM_EnvironmentBase* env)
+{
+	return releaseFreeMemoryPages(env, 0);
+}
+
+uintptr_t
+MM_MemorySubSpaceSemiSpace::releaseFreeMemoryPages(MM_EnvironmentBase* env, uintptr_t includeMemoryType)
```

Comment:
Whole this method is not really necessary. For completeness it would be fine to have. But to really complete we shold even assert that nobody passes OLD type of memory ever. 
I'd just remove it and leave the one without parameters.